src/argus_htmx/user/preferences/models.py

```diff
@@ -0,0 +1,58 @@
+from django import forms
+
+from argus.auth.models import Preferences, PreferencesManager
+
+from argus_htmx.constants import (
+    DATETIME_FORMATS,
+    DATETIME_DEFAULT,
+    DATETIME_CHOICES,
+    PAGE_SIZE_CHOICES,
+    DEFAULT_PAGE_SIZE,
+    THEME_CHOICES,
+)
+
+
+class DateTimeFormatForm(forms.Form):
+    datetime_format_name = forms.ChoiceField(required=False, choices=DATETIME_CHOICES)
+
+    def clean_datetime_format(self):
+        return self.cleaned_data.get("datetime_format", DATETIME_DEFAULT) or DATETIME_DEFAULT
+
+
+class PageSizeForm(forms.Form):
+    page_size = forms.TypedChoiceField(required=False, choices=PAGE_SIZE_CHOICES, coerce=int)
+
+    def clean_page_size(self):
+        return self.cleaned_data.get("page_size", DEFAULT_PAGE_SIZE) or DEFAULT_PAGE_SIZE
+
+
+class ThemeForm(forms.Form):
+    theme = forms.ChoiceField(choices=THEME_CHOICES)
+
+
+class ArgusHtmxPreferencesManager(PreferencesManager):
+    def get_queryset(self):
+        return super().get_queryset().filter(namespace=ArgusHtmxPreferences._namespace)
+
+    def create(self, **kwargs):
+        kwargs["namespace"] = ArgusHtmxPreferences._namespace
+        return super().create(**kwargs)
+
+
+class ArgusHtmxPreferences(Preferences):
+    _namespace = "argus_htmx"
+    FORMS = {
+        "datetime_format_name": DateTimeFormatForm,
+        "page_size": PageSizeForm,
```

Comment:
rename to `incident_list_page_size` or something akin. there will probably be other listings

Reply:
I hope not actually. There ought not be so many destinations, profiles, timeslots etc. per user that paging (or a table) is needed.

Reply:
still, can't hurt to be explicit about its purpose

Reply:
Worst case: even more settings...